packages/app/src/modules/error.js

```diff
@@ -37,21 +39,29 @@ export class EmptyResponseError extends VisualizationError {
 }
 
 export class NoSeriesError extends VisualizationError {
-    constructor() {
+    constructor(visType) {
         super(
             EmptySeries,
-            i18n.t('Series is empty'),
-            i18n.t('Add at least one item to Series.')
+            i18n.t(`{{visType}} is empty`, {
+                visType: getAxisNameByVisType(AXIS_ID_COLUMNS, visType),
+            }),
+            i18n.t('Add at least one item to {{visType}}.', {
+                visType: getAxisNameByVisType(AXIS_ID_COLUMNS, visType),
+            })
         )
     }
 }
 
 export class NoCategoryError extends VisualizationError {
-    constructor() {
+    constructor(visType) {
         super(
             EmptyCategory,
-            i18n.t('Category is empty'),
-            i18n.t('Add at least one item to Category.')
+            i18n.t(`{{visType}} is empty`, {
```

Comment:
Nitpick - this variable isn't actually the `visType` (tangent - if it were `visType` I would recommend spelling out `visualizationType` for someone not familiar with the lingo) , it's the `axisName`.  Descriptive variable names help translators get context, though this might be over the heads of many translators anyway.

Reply:
Yeah, this is a copy/paste error. Fixed. Also changed all `visType` to `visualizationType` for i18n variables.